packages/app-builder-backend/src/eject/templates/app.ts

```diff
@@ -0,0 +1,49 @@
+import { js } from './js'
+import { lint } from './format'
+
+export const generateApp = () => {
```

Comment:
Feels a bit fragile / hard to maintain. I think in the future, we should do our own CRA template similar to the foundations one that has all this stuff pre-built

Reply:
Not for today though!